torch/_dynamo/eval_frame.py

```diff
@@ -114,6 +114,43 @@ def _maybe_set_eval_frame(callback: DynamoCallback):
         return set_eval_frame(callback)
 
 
+_stance = "default"
+
+
+def _set_stance(stance: str):
+    global _stance
+    prior = _stance
+    _stance = stance
```

Comment:
Should we assert here that there isn't currently an eval frame hook set?

If there is, then the stance won't take effect until the next dynamo region is entered.

Reply:
`_dynamo_forbidden` prevents `set_stance` from being used in a `torch.compile` region. Was there another case you were thinking about?

Reply:
Calling this function after a graph break that forces a fallback to eager.